mainwindow.cpp

```diff
@@ -178,21 +184,19 @@ void MainWindow::mapping_card_value()
 
     //handle move count
     oldCardArr ^= cardArr;
-    for(; oldCardArr>0;)
+    while(oldCardArr>0)
     {
         if(oldCardArr%2)
         {
-            numberOfMove++;             // need testing for the moveCount value
             numberOfOpenCard++;
         }
         oldCardArr = oldCardArr >> 1;
         if(numberOfOpenCard >= 2)
         {
             numberOfOpenCard = 0;
-//            is_matched_cards(card1, card2); // to be added
+            uncheck_cards(1,1);
```

Comment:
I did add a comment for this line, but it seems that I have already pushed the commit before that. the (1,1) value is just a random value to call uncheck_cards() to uncheck ALL the cards. It will be updated after find_bit_index() is implemented.